Prac_06/guitar.py

```diff
@@ -0,0 +1,24 @@
+
+"""
+Word Occurrences
+Estimate: 10 minutes
+Actual:   5 minutes
+"""
+class Guitar:
+
+    def __init__(self,name="",year=0,cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+
+    def get_age(self):
+        age = 2023 - self.year
```

Comment:
the year is static but in a year this would change to 2024. I'd recommend (not necessary in this exercise) to use the datetime module